mkosi.md

```diff
@@ -41,58 +41,60 @@ whistles.
 
 The following command line verbs are known:
 
-`build`
-
-: This builds the image, based on the settings passed in on the
-  command line or read from a `mkosi.conf` file. This
-  verb is the default if no verb is explicitly specified. This command
-  must be executed as `root`. Any arguments passed after `build` are
-  passed as arguments to the build script (if there is one).
-
-`clean`
-
-: Remove build artifacts generated on a previous build. If combined
-  with `-f`, also removes incremental build cache images. If `-f` is
-  specified twice, also removes any package cache.
-
 `summary`
 
 : Outputs a human-readable summary of all options used for building an
   image. This will parse the command line and `mkosi.conf` file as it
   would do on `build`, but only output what it is configured for and not
   actually build anything.`
 
+`build`
+
+: This builds the image, based on the settings passed in on the
+  command line or read from a `mkosi.conf` file. This
+  command is the default if no verb is explicitly specified. This command
+  must be executed as `root`. Any arguments passed after the `build` verb are
+  passed as arguments to the build script (if there is one).
+
 `shell`
 
 : This builds the image if it is not built yet, and then invokes
-  `systemd-nspawn` to acquire an interactive shell prompt in it. If
-  this verb is used an optional command line may be specified which is
-  then invoked in place of the shell in the container. Combine this
-  with `-f` in order to rebuild the image unconditionally before
-  acquiring the shell, see below. This command must be executed as
-  `root`.
+  `systemd-nspawn` to acquire an interactive shell prompt in it. An optional
+  command line may be specified after the `shell` verb, to be invoked in place
+  of the shell in the container. Use `-f` in order to rebuild the image
+  unconditionally before acquiring the shell, see below. This command must be
+  executed as `root`.
 
 `boot`
 
-: Similar to `shell` but boots the image up using `systemd-nspawn`. If
-  this verb is used an optional command line may be specified which is
-  passed as "kernel command line" to the init system in the image.
+: Similar to `shell`, but boots the image using `systemd-nspawn`. An optional
+  command line may be specified after the `boot` verb, which is then passed as
+  the "kernel command line" to the init system in the image.
 
 `qemu`
 
-: Similar to `boot` but uses `qemu` to boot up the image, i.e. instead
-  of container virtualization VM virtualization is used. This verb is
+: Similar to `boot`, but uses `qemu` to boot up the image, i.e. instead
+  of container virtualization VM virtualization is used. This command is
```

Comment:
I'm not really against calling it a command (I find talking about commands and subcommands a bit clearer than talking about verbs), but the other sections talk about verbs and I think we should use the same language everywhere.

Reply:
I remove the changes to 'command' from 'verb'.